content/en/tracing/setup/dotnet-core.md

```diff
@@ -285,23 +260,57 @@ On Linux:
 ```bash
 # Set environment variables
 export DD_TRACE_AGENT_URL=http://localhost:8126
-export DD_SERVICE_NAME=MyService
-export DD_AdoNet_ENABLED=false
+export DD_ENV=prod
+export DD_SERVICE=MyService
+export DD_VERSION=abc123
+export DD_ADONET_ENABLED=false
 
 # Launch application
 dotnet example.dll
 ```
 
 {{% /tab %}}
 
+{{% tab "Code" %}}
+
+To configure the Tracer in application code, create a `TracerSettings` from the default configuration sources. Set properties on this `TracerSettings` instance before passing it to a `Tracer` constructor. For example:
+
+```csharp
+using Datadog.Trace;
+
+// read default configuration sources (env vars, web.config, datadog.json)
+var settings = TracerSettings.FromDefaultSources();
+
+// change some settings
+settings.Environment = "prod";
+settings.ServiceName = "MyService";
+settings.ServiceVersion = "abc123";
+settings.AgentUri = new Uri("http://localhost:8126/");
+
+// disable the AdoNet integration
+settings.Integrations["AdoNet"].Enabled = false;
+
+// create a new Tracer using these settings
+var tracer = new Tracer(settings);
+
+// set the global tracer
+Tracer.Instance = tracer;
+```
+
+**Note:** Settings must be set on `TracerSettings` _before_ creating the `Tracer`. Changes made to `TracerSettings` properies after the `Tracer` is created are ignored.
+
+{{% /tab %}}
+
 {{% tab "JSON file" %}}
 
 To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be a hash with a key/value pair for each setting. For example:
 
 ```json
 {
   "DD_TRACE_AGENT_URL": "http://localhost:8126",
-  "DD_SERVICE_NAME": "MyService",
+  "DD_ENV": "prod",
+  "DD_SERVICE": "MyService",
+  "DD_VERSION": "abc123",
   "DD_ADONET_ENABLED": "false"
```

Comment:
```suggestion
  "DD_ADONET_ENABLED": "true"
```

Reply:
Having `DD_ADONET_ENABLED` was just an example of how users can change the default values. All integration are already enabled by default. Feel free to remove the line entirely if it's confusing. Also, we realized recently that it's case-sensitive on Linux, so it should be `DD_AdoNet_ENABLED`.